.github/workflows/build-binaries.yaml

```diff
@@ -0,0 +1,70 @@
+---
+#################################################################################
+# Child pipeline to build hoprnet binaries
+#################################################################################
+name: Build binaries
+
+on:
+  workflow_call:
+    inputs:
+      branch:
+        required: true
+        type: string
+      runner:
+        required: true
+        type: string
+      package:
+        required: false
+        type: string
+        default: hoprd
+      binary_name:
+        required: false
+        type: string
+concurrency:
+  group: ${{ github.head_ref }}-build-binary-${{ inputs.binary_name }}
+  cancel-in-progress: true
+jobs:
+  build-binary:
+    runs-on: ${{ inputs.runner }}
+    timeout-minutes: 60
+    steps:
+      - name: Checkout hoprnet repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.branch }}
+
+      - name: Install Nix
+        uses: cachix/install-nix-action@v27
+        with:
+          github_access_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: cachix/cachix-action@v14
+        with:
+          name: hoprnet
+          authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
+        env:
+          USER: runner
+
+      - name: Build hoprd binary
+        run: |
+          nix build .#${{ inputs.package }}
+        env:
+          PKG: ${{ inputs.package }}
+
+      - name: Upload hoprd binary
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ inputs.binary_name }}
+          path: ${{ github.workspace }}/result/bin/hoprd
+
+    # - name: Build hopli binary
+    #   if: contains(github.event.pull_request.labels.*.name, 'component:hopli')
+    #   run: |
+    #     nix build .#hopli-${{ inputs.system }}
+
+    # - name: Upload hopli binary
+    #   if: contains(github.event.pull_request.labels.*.name, 'component:hopli')
+    #   uses: actions/upload-artifact@v4
+    #   with:
+    #     name: hopli-${{ inputs.system }}
+    #     path: ${{ github.workspace }}/result/bin/hopli
```

Comment:
hopli does not need to be multiplatform, or does it?

Reply:
It does.

Reply:
But that is a simple devtool. Who else is using it?